### PR TITLE
adding angular implementation to directory 2-templating

### PR DESCRIPTION
**Pull request by ismailassa**
adding angular implementation to the directory 2-templating

### CODE REVIEW

File: content/2-templating/6-conditional/angular/trafficlight.component.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  @Input() lightIndex:number = 0;
+
+  nextLight():void{
+    if (this.lightIndex + 1 > TRAFFIC_LIGHTS.length - 1) {
+			this.lightIndex = 0
+		} else {
+			this.lightIndex++
+		}
+  }
+
```
Comment (matschik) [inline content/2-templating/6-conditional/angular/trafficlight.component.ts:38]: Run "npm run format" for better indentation

File: content/2-templating/2-styling/angular/cssstyle.component.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-helloworld',
+  template:`
+    <h1 class="title">Hello world</h1>
+    <button style="font-size:10rem">I am a button</button>
+  `,
```
[the remark this answers is not in the document]
Comment (ismailassa) [reply content/2-templating/2-styling/angular/cssstyle.component.ts:8]: That's an interesting remark that would maybe require to change the display of an angular element.
What do you suggest?  
Maybe we could make separate tabs for the component files (html css and ts) or just each file below each other.

File: content/2-templating/2-styling/angular/cssstyle.component.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  styles:[" .title {color: red} "]
```
[the remark this answers is not in the document]
Comment (ismailassa) [reply content/2-templating/2-styling/angular/cssstyle.component.ts:9]: ok

**Review by ismailassa — COMMENTED** [review]
feedback implemented

**Pull request merged**